src/main/java/games/strategy/engine/framework/GameRunner.java

```diff
@@ -540,21 +540,16 @@ public static void exitGameIfFinished() {
    * todo, replace with something better
    * Get the chat for the game, or null if there is no chat.
    */
-  public static Chat getChat() {
+  public static Optional<Chat> getChat() {
     final ISetupPanel model = setupPanelModel.getPanel();
     if (model instanceof ServerSetupPanel) {
-      return model.getChatPanel().getChat();
+      return Optional.ofNullable(model.getChatPanel().getChat());
     } else if (model instanceof ClientSetupPanel) {
-      return model.getChatPanel().getChat();
+      return Optional.ofNullable(model.getChatPanel().getChat());
     } else {
-      return null;
+      return Optional.empty();
     }
   }
-
-  public static boolean hasChat() {
-    return getChat() != null;
-  }
-
   /**
```

Comment:
Checkstyle: "'METHOD_DEF' should be separated from previous statement."

Reply:
fixed in 826fb08